articles/blockchain-as-a-revolution-inhealth/index.md

```diff
@@ -0,0 +1,86 @@
+
+## Advantages of Blockchain in Healthcare 
+***Provenance & Increased Data Integrity:*** [Transactions in the Blockchain network are immutable](https://medium.com/pikciochain/how-is-blockchain-revolutionizing-healthcare-7f6d2a48e561). This means that once a transaction is approved, it cannot be altered or deleted. Transactions are shared across the network and securely recorded. The data integrity is safe, and time-stamp can be verified by any user. The original author of a transaction in a blockchain network is recorded and remains accessible for the entire life of the blockchain.
+
+***Security:*** Transactions on the blockchain network must first be verified by other blockchain users before they are recorded. Once verified, the transactions are encrypted and linked to a previous transaction using cryptography. [The blockchain is virtually impossible to hack the because  data is stored across a network of devices](https://medium.com/pikciochain/how-is-blockchain-revolutionizing-healthcare-7f6d2a48e561).
+
+***Transparency:*** Before an existing version is changed, everyone in the blockchain network must agree since the records are shared. An overhaul of the whole network would be required to change a single transaction on a blockchain network, making the data on the network transparent, consistent and largely accurate. All participants can access any transaction on the blockchain as long as they have a private key, as every transaction is decentralized. This makes it easier to share information across multiple platforms.
+
+### Implementing Blockchain in health care
+![Health care img](/engineering-education/blockchain-as-a-revolution-inhealthcare/Healthcare-img.jpg)
+[*Image Source: https://www.gettyimages.com/detail/photo/nurse-and-surgeon-inspecting-screens-in-operating-royalty-free-image/1202320470*](https://www.gettyimages.com/detail/photo/nurse-and-surgeon-inspecting-screens-in-operating-royalty-free-image/1202320470)
+
+There are numerous applications of Blockchain in healthcare:
+
+### Record Keeping
+***Medical History Tracking***
+With Blockchain, patients' treatment history can be tracked anywhere and at any time with no ambiguity, hence providing health practitioners a better context.
+
+_In Data Security: [Do you know that your medical data is worth more than your credit card number in the black market?](https://medium.com/pikciochain/how-is-blockchain-revolutionizing-healthcare-7f6d2a48e561) According to a report by Protenus Breach Barometer, about 140 million patient health records were hacked between 2015 and 2016._
+>**Blockchain has the potential to keep private information safe and secure while still being accessible anywhere.**
```

Comment:
Including security details that make Blockchain uniquely suited to provide more explanation and avoid sounding like an infomercial.

Reply:
Can you please explain what you really want me to do here. @sudosoph